Directory.Build.targets

```diff
@@ -1,21 +1,23 @@
 <Project>
 
   <PropertyGroup>
-    <UseUWP Condition="$(TargetFramework.Contains(`uap10.0`)) or '$(TargetFramework)' == 'net461'">true</UseUWP>
+    <!-- Exclude Notifications project from this since it sets different min versions than what we want for notifications -->
+    <UseUWP Condition="($(TargetFramework.StartsWith('uap10.0')) or '$(TargetFramework)' == 'net461') and '$(MSBuildProjectName)' != 'Microsoft.Toolkit.Uwp.Notifications'">true</UseUWP>
   </PropertyGroup>
 
   <Choose>
     <!-- We'll include signing the Notifications library since we need the DLL signature to match for interop from class libraries to main projects -->
-    <When Condition="(!$(TargetFramework.Contains(`uap10.0`)) and '$(TargetFramework)' != 'native' and '$(IsSampleProject)' != 'true') or $(MSBuildProjectName) == 'Microsoft.Toolkit.Uwp.Notifications'">
+    <When Condition="!($(TargetFramework.StartsWith('uap10.0')) or '$(TargetFramework)' == 'native' or '$(IsSampleProject)' == 'true') or '$(MSBuildProjectName)' == 'Microsoft.Toolkit.Uwp.Notifications'">
       <PropertyGroup>
         <SignAssembly>true</SignAssembly>
         <AssemblyOriginatorKeyFile>$(MSBuildThisFileDirectory)toolkit.snk</AssemblyOriginatorKeyFile>
       </PropertyGroup>
     </When>
   </Choose>
 
-  <!--Exclude Notifications project from this since it sets different min versions than what we want for notifications-->
-  <Import Project="$(MSBuildThisFileDirectory)build\Windows.Toolkit.Uwp.Build.targets" Condition="'$(UseUWP)' == 'true' and $(MSBuildProjectName) != 'Microsoft.Toolkit.Uwp.Notifications'" />
+  <Import Project="$(BuildToolsDirectory)Windows.Toolkit.Uwp.Build.targets" Condition="'$(UseUWP)' == 'true'" />
+
+  <Import Project="$(BuildToolsDirectory)Windows.Toolkit.Workarounds.Xaml.targets" Condition="'$(IsCoreProject)' == 'true' and '$(GenerateDocumentationFile)' == 'true'" />
```

Comment:
Isn't GenerateDocumentationFile always true when IsCoreProject is true?
Based on this:
https://github.com/windows-toolkit/WindowsCommunityToolkit/pull/3894/files#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eR15-R17

Reply:
This is in targets, so, could be overriden in the project file (_e.g.: App projects_).

Besides, the warnings surface only when we generate `xml` doc from a project containing `xaml` generated souces. So, we only need this workaround "strictly" in those scenarios.

App projects don't need to generate `xml` doc as opposed to library projects, so we don't need to include this workaround for those projects.

Reply:
Not necessarily! This is in targets, so, could be overriden in the project file (_e.g.: App projects_).

Besides, the warnings surface only when we generate `xml` doc from a project containing `xaml` generated souces. So, we only need this workaround "strictly" in those scenarios.

App projects don't need to generate `xml` doc as opposed to library projects, so we don't need to include this workaround for those projects.

Reply:
I thought we don't need to generate `XML` doc for App projects but the doc file is used by `StyleCop` to analyze the `XML` comments!

I reverted this for now.

BTW, `StyleCop` Analyzer have direct access to source, IDK why they need a separate XML file for analysis!?

Reply:
I removed the extra check. I thought we don't need to generate `XML` doc for App projects but the doc file is used by `StyleCop` to analyze the `XML` comments! BTW, `StyleCop` Analyzer have direct access to source, IDK why they need a separate XML file for analysis!?